docs/excel/custom-functions-dynamic-arrays.md

```diff
@@ -58,6 +60,33 @@ function spillRectangle() {
 }
 ```
 
+The fourth example shows how to return a dynamic spill array from a streaming function. The results spill down, like the first example, and increment once a second based on the `amount` parameter. To learn more about streaming functions, see [Make a streaming function](custom-functions-web-reqs.md#make-a-streaming-function).
+
+```javascript
+    /** @CustomFunction
+     * @description Increment the cells with a given amount every 1000 milliseconds.
+     * @param {number} amount The amount to add to the cell value on each increment.
+     * @param {CustomFunctions.StreamingInvocation<number[][]>} invocation Parameter to send results to Excel or respond to the user canceling the function. A dynamic array.
+     * @returns An incrementing value. Returns a dynamic spilled array with multiple results.
+     */
+    function increment(amount: number, invocation: CustomFunctions.StreamingInvocation<number[][]>): void {
+      let firstResult = 0;
+      let secondResult = 1;
+      let thirdResult = 2;
+
+      const timer = setInterval(() => {
+        firstResult += amount;
+        secondResult += amount;
+        thirdResult += amount;
+        invocation.setResult([[firstResult], [secondResult], [thirdResult]]);
+      }, 1000);
+
+      invocation.onCanceled = () => {
```

Comment:
This requires the @cancelable attribute in the code section above. I'm not sure how to test this though to see if it really cancels.

Reply:
My understanding of our documentation is that @cancelable is not relevant for streaming functions: 
<img width="697" alt="image" src="https://user-images.githubusercontent.com/7977793/206058536-e7db03bf-6c5e-4c31-801e-9504d722ed00.png">

Screenshot is from this article section: https://learn.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-web-reqs#cancel-a-function

Reply:
I based this new streaming code sample off the streaming sample in this article: https://learn.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-web-reqs#make-a-streaming-function, which has an onCanceled method and doesn't include @cancelable in the JS doc. Let me know if the existing snippet needs to be updated, too

Reply:
I clarified this note based on our offline conversation and using information from https://learn.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json-autogeneration#cancelable

Ready for you to take another look. Thanks!